src/Uno.UWP/Globalization/NumberFormatting/NumeralSystemTranslator.cs

```diff
@@ -0,0 +1,169 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Uno.Globalization.NumberFormatting;
+
+namespace Windows.Globalization.NumberFormatting
+{
+	public partial class NumeralSystemTranslator
+	{
+		private string _numeralSystem;
+
+		public NumeralSystemTranslator() : this(new string[] { "en-US" })
+		{
+
+		}
+
+		public NumeralSystemTranslator(IEnumerable<string> languages)
+		{
+			ValidateLanguages(languages);
+
+			Languages = languages.ToList();
+			ResolvedLanguage = NumeralSystemTranslatorHelper.GetResolvedLanguage(Languages[0]);
+			NumeralSystem = NumeralSystemTranslatorHelper.GetNumeralSystem(Languages[0]);
+		}
+
+		public string NumeralSystem
+		{
+			get => _numeralSystem;
+			set
+			{
+				if (value is null)
+				{
+					throw new ArgumentNullException("Value cannot be null.");
+				}
+
+				_numeralSystem = NumeralSystemTranslatorHelper.ToPascalCase(value);
+			}
+		}
+
+		public IReadOnlyList<string> Languages { get; }
+
+		public string ResolvedLanguage { get; }
+
+		private void ValidateLanguages(IEnumerable<string> languages)
+		{
+			if (languages is null)
+			{
+				throw new NullReferenceException("Invalid pointer");
+			}
+
+			if (!languages.Any())
+			{
+				throw new ArgumentException("The parameter is incorrect.");
+			}
+
+			foreach (var language in languages)
+			{
+				NumeralSystemTranslatorHelper.GetNumeralSystem(language);
+			}
+		}
+
+		public string TranslateNumerals(string value)
+		{
+			if (NumeralSystem.Equals("Arab", StringComparison.InvariantCultureIgnoreCase) ||
+				NumeralSystem.Equals("ArabExt", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
I'm not sure if this should be Ordinal or InvariantCulture.

Reply:
I just googled it and I think it should be Ordinal. NumeralSystem is always pascal case so we can perform case-sensitive comparison. @MartinZikmund Do you agree?

Reply:
Yes it seems like `Ordinal` should be enough